src/south/ProtocolHandler.class.js

```diff
@@ -101,9 +110,26 @@ class ProtocolHandler {
    * @return {void}
    */
   addValues(values) {
+    const flush = async (flag = 'time') => {
+      this.logger.silly(`${flag}: ${this.buffer.length}, ${this.dataSource.dataSourceId}`)
+      await this.engine.addValues(this.dataSource.dataSourceId, this.buffer)
+      this.buffer = []
+      clearTimeout(this.bufferTimeout)
```

Comment:
Need to check if bufferTimeout is null. Can happen if more than BUFFER_MAX values are retrieved at first call of this function